docs/tutorial/using-checkbox/test-report.rst

```diff
@@ -0,0 +1,129 @@
+.. _test-report:
+
+Review Test Report 
+==================
+
+Once you finish your first test run, you may want to review the test reports. 
+Checkbox generates test reports in different formats that can be used to 
+easily share the results of a test session. By default, a :ref:`text summary<
+text-summary>` and :ref:`submission files<submission-files>` are created. This 
+section will provide you with information on where to locate the report files 
+and their respective contents.
+
+.. note::
+
+    You can tailor desired reports in your launcher file, or define your own 
+    exporter to customize the reports. See :doc:`launcher<../../reference/
```

Comment:
this link, fix it the same way `WARNING: unknown document: '../../reference/ launcher'`

Reply:
fixed in [6159a36](https://github.com/canonical/checkbox/pull/637/commits/6159a3674bc44d689c7a6ab7acda85d5f68d13d1)